Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.19-openshift-4.14 AS builder
+FROM registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.20-openshift-4.14 AS builder
```

Comment:
is there a rhel-9 base instead now?

Reply:
there is one but I don't think any of the operators have updated them yet.